client/src/components/QandA/QuestionListEntry.jsx

```diff
@@ -9,23 +9,23 @@ class QuestionListEntry extends React.Component {
     super(props);
     this.state = {
       answersShown: 2,
-      helpfulClicked: false,
-      question_helpfulness: this.props.data.question_helpfulness
+      questionHelpfulClicked: false,
+      questionHelpfulCount: this.props.data.question_helpfulness
     };
     this.showMoreAnswers = this.showMoreAnswers.bind(this);
     this.showLessAnswers = this.showLessAnswers.bind(this);
   }
 
   markQuestionHelpful(question_id) {
-    this.state.helpfulClicked ? console.log('You can only click once') :
+    this.state.questionHelpfulClicked ? console.log('You can only click once') :
```

Comment:
If you meant to leave this message to the user, I suggest changing the console.log to an alert.